invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/requests.js

```diff
@@ -0,0 +1,279 @@
+  return (
+    total && (
+      <Grid>
+        <Grid.Row>
+          <Grid.Column width={16}>
+            <Segment>
+              <Grid>
+                <Grid.Row
+                  verticalAlign="middle"
+                  className="small padding-tb-5 deposit-result-header"
+                >
+                  <Grid.Column width={4}>
+                    <Count
+                      label={() => (
+                        <>
+                          {total} {i18next.t("result(s) found")}
+                        </>
+                      )}
+                    />
+                  </Grid.Column>
+                  <Grid.Column
+                    width={12}
+                    textAlign="right"
+                    className="padding-r-5"
+                  >
+                    {sortOptions && (
+                      <Sort
+                        values={sortOptions}
+                        label={(cmp) => (
+                          <>
+                            {i18next.t("Sort by")} {cmp}
+                          </>
+                        )}
+                      />
+                    )}
+                  </Grid.Column>
+                </Grid.Row>
+                <Grid.Row>
+                  <Grid.Column>
+                    <ResultsList />
+                  </Grid.Column>
+                </Grid.Row>
+              </Grid>
+            </Segment>
+          </Grid.Column>
+        </Grid.Row>
+        <Grid.Row verticalAlign="middle">
+          <Grid.Column width={4}></Grid.Column>
+          <Grid.Column width={8} textAlign="center">
+            <Pagination
+              options={{
+                size: "mini",
+                showFirst: false,
+                showLast: false,
+              }}
+            />
+          </Grid.Column>
+          <Grid.Column textAlign="right" width={4}>
+            <ResultsPerPage
+              values={paginationOptions.resultsPerPage}
+              label={(cmp) => (
+                <>
+                  {" "}
+                  {cmp} {i18next.t("results per page")}
+                </>
+              )}
+            />
+          </Grid.Column>
+        </Grid.Row>
+      </Grid>
+    )
+  );
+};
+
+export function RequestsResultsGridItemTemplate({ result, index }) {
+  return (
+    <Card fluid key={index} href={`/requests/${result.metadata.id}`}>
+      <Card.Content>
+        <Card.Header>{result.metadata.title}</Card.Header>
+        <Card.Description>
+          <div
+            dangerouslySetInnerHTML={{ __html: result.metadata.description }}
+          />
+        </Card.Description>
+      </Card.Content>
+    </Card>
+  );
+}
+
+export function RequestsResultsItemTemplate({ result, index }) {
+  return (
+    <Item key={index}>
+      <Item.Content>
+        <Item.Header href={`/requests/${result.id}`}>
+          {result.metadata.title}
+          {result.metadata.type && (
+            <span style={{ marginLeft: "5px" }}>
```

Comment:
inline style, maybe it is possible to use a class similar to `padding-r-5`

Reply:
Will rework on it in my next PR, regarding communities tab :)